subsys/mpsl/mpsl_init.c

```diff
@@ -118,6 +121,125 @@ static uint8_t m_config_clock_source_get(void)
 #endif
 }
 
+#if CONFIG_MPSL_FEM_NRF21540_GPIO
```

Comment:
I think you should use `#ifdef` for `bool` options from Kconfig

Reply:
Good point. I used `#if IS_ENABLED(.)` for `bool`